style/modules/scrollCarousel.css

```diff
@@ -0,0 +1,24 @@
+.scrollCarousel {
+    max-height: 300px;
+    overflow: hidden;
+}
+
+.scrollCarousel--contents {
+    width: max-content;
+    display: flex;
+    animation-direction: alternate;
+    animation-name: moveSlideshow;
+    animation-duration: 5s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+}
+
```

Comment:
I just tried it with images and realized we'll probably want some spacing between the images. for that, probably easiest to add a section here that's like

```css
.scrollCarousel--contents > * {
  margin-left: ...;
  margin-right: ...;
}
```

Reply:
(resolved this on line 18)